leaflet-treering.js

```diff
@@ -2743,6 +2806,173 @@ function ImageAdjustment(Lt) {
 
 }
 
+/**
+* Change measurement options (set subAnnual, previously hasLatewood, and direction)
+* @constructor
+* @param {Ltreeing} Lt - Leaflet treering object
+*/
+function MeasurementOptions(Lt) {
+  this.choice = false;
+  this.btn = new Button(
+    'timeline',
+    'Change measurement direction and annual/sub-annual mode',
+    () => { Lt.disableTools(); this.enable() },
+    () => { this.disable() }
+  );
+
+  /**
+  * Data from Lt.preferences
+  * @function preferencesInfo
+  */
+  MeasurementOptions.prototype.preferencesInfo = function () {
+    if (Lt.preferences.forwardDirection == false) { // direction object
+      this.forwardDirection = false;
+    } else {
+      this.forwardDirection = true;
+    }
+
+    let ewFalse = [];
+    for (i = 0; i <= Lt.data.points.length; i++) { // for assets which use hasLatewood
```

Comment:
change this over to a filter
ewFalse = Lt.data.points.filter(p => return (Lt.data.points[i] && Lt.data.points[i].earlywood == false)); 
if(ewFalse.length > 0) {
}